grakn-migration/base/src/main/java/ai/grakn/migration/base/AbstractMigrator.java

```diff
@@ -69,7 +101,7 @@ public AbstractMigrator registerMacro(Macro macro){
             return Optional.of(queryBuilder.parseTemplate(template, data));
         } catch (GraqlTemplateParsingException e){
             LOG.warn("Query was not sent to loader- " + e.getMessage());
-            LOG.warn("See the Grakn engine logs for more detail about loading status and any resulting stacktraces: " + properties.getLogFilePath());
+            LOG.warn("See the Grakn engine logs for more detail about loading status and any resulting stacktraces");
```

Comment:
I am still not happy with this. If migration fails I should not have to log into the server to discover why. I should be told immediately. A usability issue which I think still needs to be addressed.